resin/README.md

```diff
@@ -65,10 +65,11 @@ Need help? Contact [Datadog support][7].
 
 
 [1]: https://caucho.com/
-[2]: https://github.com/DataDog/integrations-core/blob/master/resin/datadog_checks/resin/data/conf.yaml.example
+[2]: https://github.com/DataDog/integrations-extras/tree/master/resin
```

Comment:
Something a little strange still going on... link 2 is referenced on line 11, according to that it should link to the Datadog Agent package, and that's not where this link is going.

Reply:
```suggestion
[2]: https://app.datadoghq.com/account/settings#agent
```